src/test/suite/participant/participant.test.ts

```diff
@@ -1026,7 +1033,7 @@ suite('Participant Controller Test Suite', function () {
             const chatResult = await invokeChatHandler(chatRequestMock);
             const askForDBMessage = chatStreamStub.markdown.getCall(0).args[0];
             expect(askForDBMessage).to.include(
-              'What is the name of the database you would like this query to run against?'
+              'Which database would you like this query to run against? Select one by either clicking on an item in the list or typing the name manually in the chat.\n\n'
```

Comment:
@nirinchev @Anemy I combined the message used by `handleEmptyNamespaceMessage` and regular namespace prompting. It's nice to mention the option to type it manually right away and just repeat the same exact prompt if they send an empty message, that also lets us re-use the helper.